src/main/kotlin/pl/edu/agh/gem/internal/model/expense/AcceptedExpense.kt

```diff
@@ -0,0 +1,37 @@
+package pl.edu.agh.gem.internal.model.expense
+
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.time.Instant
+
+data class AcceptedExpense(
+    val creatorId: String,
+    val title: String,
+    val totalCost: BigDecimal,
+    val baseCurrency: String,
+    val targetCurrency: String?,
```

Comment:
do we want to have one class for pair currency and exchange rate to remove double nullability

Reply:
fixed also added that in expense manager (totalCost, baseCurrency) => amount, (targetCurrency, exchangeRate) => fxData